mods/ts/rules/critters.yaml

```diff
@@ -0,0 +1,119 @@
+DOGGIE:
+	Inherits@1: ^Infantry
+	Inherits@2: ^RegularInfantryDeath
+	Inherits@3: ^HealsOnTiberium
+	Tooltip:
+		Name: Tiberian Fiend
+	Health:
+		HP: 250
+		Shape: Circle
+			Radius: 213
+	Selectable:
+		Bounds: 24,24
+	Valued:
+		Cost: 100
+	Armor:
+		Type: Light
+	RevealsShroud:
+		Range: 4c0
+	Mobile:
+		Speed: 113
+	Voiced:
+		VoiceSet: Fiend
+	Targetable:
+		TargetTypes: Ground
+	Armament:
+		Weapon: FiendShard
+	AttackFrontal:
+		Voice: Attack
+	AttackWander:
+		WanderMoveRadius: 2
+		MinMoveDelay: 200
+		MaxMoveDelay: 600
+	-SpawnActorOnDeath@FLAMEGUY:
+	WithDeathAnimation@fire:
+		DeathSequence: die-
+		DeathTypes:
+			FireDeath: burning
+
+VISC_SML:
+	Inherits: ^Visceroid
+	Tooltip:
+		Name: Baby Visceroid
+	Health:
+		HP: 200
+	AttackWander:
+		WanderMoveRadius: 2
+		MinMoveDelay: 30
+		MaxMoveDelay: 60
+	RenderSprites:
+		Image: vissml
+
+VISC_LRG:
+	Inherits: ^Visceroid
+	Inherits@CRATESTATS: ^CrateStatModifiers
+	Tooltip:
+		Name: Adult Visceroid
+	Health:
+		HP: 500
+	Armor:
+		Type: Heavy
+	RevealsShroud:
+		Range: 4c0
+	Armament:
+		Weapon: SlimeAttack
+		FireDelay: 10
+	AutoTarget:
+		ScanRadius: 5
+	AttackFrontal:
+		Voice: Attack
+	AttackWander:
+		WanderMoveRadius: 2
+		MinMoveDelay: 25
+		MaxMoveDelay: 50
+	WithAttackAnimation:
+		AttackSequence: attack
+	Mobile:
+		Crushes: crate, infantry
+	RenderSprites:
+		Image: vislrg
+
+JFISH:
+	Inherits: ^Visceroid
```

Comment:
The selection rectangle will need to shift up:
![screen shot 2017-04-22 at 13 11 42](https://cloud.githubusercontent.com/assets/167819/25304351/5b5426da-275d-11e7-8cbb-d82faa983541.png)